domains/pele.json

```
@@ -1,10 +1,10 @@
 {
   "owner": {
-    "username": "Pele",
+    "username": "Pele12324",
     "email": "pele.puric@gmail.com",
     "discord": "483261654818226186"
   },
   "record": {
-    "A": "http://38.242.242.224/"
+    "A": "38.242.242.224"
```

A needs to be array-ed, like i talked to you before
```suggestion
    "A": ["38.242.242.224"]
```